tests/views/test_views.py

```diff
@@ -30,6 +31,26 @@ def test_join_github_form(self):
                 redirect = True
         self.assertEqual(redirect, True)
 
+    def test_join_selection_digital_justice_user(self):
+        with self.app.test_client() as client:
+            with client.session_transaction() as sess:
+                sess['email'] = 'user@digital.justice.gov.uk'
+                sess['org_selection'] = ['ministryofjustice']
+
+            response = client.get("/join-selection")
+            self.assertEqual(response.status_code, 200)
+            self.assertIn('Using Single Sign-On', str(response.data))
+
+    def test_join_selection_other_user(self):
+        with self.app.test_client() as client:
+            with client.session_transaction() as sess:
+                sess['email'] = 'user@example.com'
```

Comment:
What is meant to happen here? Ah this is not @digital.justice.gov.uk email but could be @justice.gov.uk or any other on the allowed list. The test makes it look like any email could get that far maybe
```suggestion
                sess['email'] = 'user@allowed.but.not.digital.justice.gov.uk'
```
maybe that's going too far 😂

Reply:
This test will make more sense when the other paths are padded out.